tests/ClientTest.php

```diff
@@ -154,4 +144,27 @@ public function testTokenNoAuth()
     $result = $client->validateToken();
     $this->assertNotEmpty($result);
   }
+
+  public function testApiKey()
+  {
+    $client = Client::create(
+      [
+        'api_key' => 'd699daaeeb9a4303bee8dd6d3652ee05',
+        'live' => false
+      ]
+    );
+    $rides = $client->getRideClient()->mine();
+    $this->assertNotEmpty($rides);
+  }
+
+
+  public function testApiKeyException()
+  {
+    $this->expectException(InvalidArgumentException::class);
+    $this->expectExceptionMessage('Key app_id is missing');
```

Comment:
C'est une api_key et tu envois l'erreur 'Key app_id is missing' :) Il faut mieux faire la validation dans le constructeur